src/App.tsx

```diff
@@ -27,6 +28,30 @@ function App() {
                 >
                     Learn React
                 </a>
+                <Router>
+                    <ul>
+                        <li><Link to="/complextree"> Comlex Tree </Link></li>
+                        <li><Link to="/simple"> Simple Tree </Link></li>
+                    </ul>
+                    <Routes>
```

Comment:
can you also change the paths to
/tree/complex and /tree/simple (to use nested routes)?